docs/sources/alerting/manage-notifications/view-alert-groups.md

```diff
@@ -4,49 +4,103 @@ aliases:
+By default, Grafana Alerting groups similar firing alerts (or alert instances) to prevent notification overload. For details on how notification grouping works, refer to [Group alert notifications](ref:grouping).
 
-## View alert groups
+In the Groups view, you can see alert groups, check the state of their notifications, and also filter for alert instances that match specific criteria. This view is useful for debugging and verifying your grouping of notification policies.
+
+## View alert groups and notification state
 
 To view alert groups, complete the following steps.
 
-1. In the left-side menu, click **Alerts & IRM** and then **Alerting**.
-1. Click **Groups** to view the list of existing groups.
-1. From the **Alertmanager** dropdown, select an external Alertmanager as your data source. By default, the `Grafana` Alertmanager is selected.
-1. From **Custom group by** dropdown, select a combination of labels to view a grouping other than the default. This is useful for debugging and verifying your grouping of notification policies.
+1. Click **Alerts & IRM** -> **Alerting**.
+1. Click **Groups** to view the list of groups firing notifications.
+
+   By default, alert groups are grouped by the notification policies grouping.
+
+   Each group displays its label set, contact point, and the number of alert instances (or alerts).
+
+   Then, click on a group to access its alert instances. You can find alert instances by their label set and view their notification state.
+
+### Notification states
+
+The notification state of an alert instance can be in one of the following states:
+
+- **Unprocessed**: The alert is received but its notification has not been processed yet.
+- **Suppressed**: The alert has been silenced.
+- **Active**: The alert notification has been handled. The alert is still firing and continues to be managed.
+
+### Filter alerts
+
+You can filter by label, state, or Alertmanager:
+
+- **By label**: In **Search**, enter an existing label to view alerts matching the label. For example, `environment=production,region=~US|EU,severity!=warning`.
+
+- **By state**: In **States**, select from Active, Suppressed, or Unprocessed states to view alerts matching your selected state. All other alerts are hidden.
+
+- **By Alertmanager**: In the **Alertmanager** dropdown, select an [external Alertmanager](ref:alertmanager) to view only alert groups for that specific Alertmanager. By default, the `Grafana` Alertmanager is selected.
+
+### Custom group
+
+From **Custom group by** dropdown, select a combination of labels to view a grouping other than the default. This helps validate the [grouping of your notification policies](ref:grouping).
 
 If an alert does not contain labels specified either in the grouping of the default policy or the custom grouping, then the alert is added to a catch all group with a header of `No grouping`.
 
-## Filter alerts
+## View notification errors
```

Comment:
This section seems outdated, can we update this to reflect the new contact points page?

Reply:
addressed by https://github.com/grafana/grafana/pull/89461/commits/8d3cacf22d41182e6b8e3a3a9415308d47d25a8f